docs/content/Getting-Started/Overview.mdx

```diff
@@ -8,15 +8,22 @@ isDisableFeedbackBlock: true
 
 Choose a platform to get started with below:
 
+<Grid cols={2}>
+  <GridItem
+    url="https://cubecloud.dev/auth/signup"
+    imageUrl="https://raw.githubusercontent.com/cube-js/cube.js/master/docs/src/components/DocsSwitcher/cube-logo-cloud.svg"
+    title="Cube Cloud"
+  />
+</Grid>
+
```

Comment:
@ivan-vdovin The grid appears to be broken on the Netlify Deploy Preview:

![image](https://user-images.githubusercontent.com/1781985/148986701-62c13e25-70ff-4cdf-a305-07fa9048e505.png)

Can you wrap all the `<GridItems>` inside the `<Grid` component please? Thanks in advance